### ISSUE TITLE
OpenProject reports FILE ACCESS ERROR when it fails to find the global user setting file

### ISSUE THREAD

**Issue by e9gille** [aplteam/Cider#23]
The error reported is caused by Cider looking for a template file in the wrong place. This only affects new users as previous versions of Cider created the file for the user correctly. To replicate you must first remove the user settings and then try to open a project.

```
      ]cider.openProject .
 Loading project from . 
 Checking parameters... 
   All fine 
 Bringing in the source code... 
   Code established, "watch" is "ns" 
* Command Execution Failed: FILE ACCESS ERROR: C:/Users/e9gil/Documents/MyUCMDs/Cider/.cider/config.json: Invalid file or 
      directory ("The system cannot find the path specified.")
```

**Comment by aplteam** [comment]
Fixed in 0.24.0

**Comment by aplteam** [comment]
Fix works under Windows but not on other platforms.

**Comment by aplteam** [comment]
Fully fixed in 0.24.1
